test/e2e/network/service.go

```diff
@@ -1723,9 +1724,6 @@ var _ = SIGDescribe("Services", func() {
 			jig.SanityCheckService(svc, v1.ServiceTypeLoadBalancer)
 			framework.ExpectEqual(e2eservice.GetIngressPoint(lbIngress), internalStaticIP)
 		}
-
-		ginkgo.By("switching to ClusterIP type to destroy loadbalancer")
-		jig.ChangeServiceType(svc.Namespace, svc.Name, v1.ServiceTypeClusterIP, createTimeout)
```

Comment:
Note: we won't need this extra step if we simply wait for service deletion in the end.